src/plugins/lowMach.cpp

```diff
@@ -54,8 +55,66 @@ void lowMach::setup(nrs_t* nrs, dfloat gamma)
   platform->options.setArgs("LOWMACH", "TRUE"); 
 }
 
-// qtl = 1/(rho*cp*T) * (div[k*grad[T] ] + qvol)
 void lowMach::qThermalIdealGasSingleComponent(dfloat time, occa::memory o_div)
+{
+  nrs_t* nrs = the_nrs;
+  cds_t* cds = nrs->cds;
+  mesh_t* mesh = nrs->meshV;
+  linAlg_t * linAlg = platform->linAlg;
+  
+  expansionCoeff = 0; 
+  qThermalRealGasSingleComponent(time,o_div,cds->o_S);
+  
+  qThermal = 1;
```

Comment:
This has to go before  qThermalRealGasSingleComponent() like it was before.

Reply:
qThermal = 1 must be here, because  qThermal is set to 0 at the end of  qThermalRealGasSingleComponent.